docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.7"
+services:
+
+  app:
+    build:
+      context: .
+    environment:
+      PYTHONUNBUFFERED: 1
+    volumes:
+      - .:/app
```

Comment:
What does this top-level `docker-compose.yml` get used for in practice? I wasn't able to figure out how it relates to the other changes here.

Reply:
ahah, good catch, this should actually be the subject of another PR.  I'll remove it from this one.

I "dockerfied" my local copy of this repo rather than using a virtualenv to install the cookiecutter package